docs/extensions/logging-and-monitoring.md

```diff
@@ -90,31 +97,35 @@ data:
-Also, as fluent-bit parses the logs based on the container name you should make sure that the container names inside your provider-specific pods are prefixed with your extension name.
+Logs can be produced from various sources, such as containers or systemd, and in different formats. The fluent-bit design supports configurable [data pipeline](https://docs.fluentbit.io/manual/concepts/data-pipeline) to address that problem. Gardener provides such [configuration](https://github.com/gardener/gardener/blob/master/charts/seed-bootstrap/charts/fluentd-es/templates/fluent-bit-configmap.yaml) for logs produced by all its core managed components as a `ConfigMap`. Extensions can contribute their own, specific configurations as `ConfigMap`s too. See for example the [logging configuration](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/gardener-extension-provider-aws/templates/configmap-logging.yaml) for the Gardener AWS provider extension. The Gardener reconciliation loop watches such resources and updates the fluent-bit agents dynamically.
 
-### Submitting logging parse configuration?
+### Extensions logging integration
 
-To submit a new logging parse configuration, create a `ConfigMap` labeled `extensions.gardener.cloud/configuration=logging` with a single data field - `filter-kubernetes.conf`, containing the fluent-bit configuration for parsing container logs. Before deploying the central fluent-bit instances into the `garden` namespace, Gardener will read all `ConfigMap`s in `garden` with that label and apply the configuration encoded in them.
-Such `ConfigMap`s may contain a single field in their `data`:
+To integrate with Gardener logging, extensions can and *should* specify how fluent-bit will handle the logs produced by the managed components that they contribute to Gardener. Normally, that would require to configure a *parser* for the specific logging format, if none of the available is applicable, and a *filter* defining how to apply it. For a complete reference for the configuration options, refer to fluent-bit's [documentation](https://docs.fluentbit.io/manual/). 
+**Note:** At the moment only *parser* and *filter* configurations are supported.
 
-**Example:** The `Worker` controller might deploy a `machine-controller-manager` into the shoot namespace, and it wants to submit some logging parse configuration.
+To contribute its own configuration to the fluent-bit agents data pipelines, an extension must provide it as a `ConfigMap` labeled `extensions.gardener.cloud/configuration=logging` and deployed in the seed's `garden` namespace. Unlike the monitoring stack, where configurations are deployed per shoot, here a *single* configuration `ConfigMap` is sufficient and it applies to all fluent-bit agents in the seed. Its `data` field can have the following properties:
+- `filter-kubernetes.conf` - configuration for data pipeline [filters](https://docs.fluentbit.io/manual/concepts/data-pipeline/filter)
```

Comment:
Please, add some lines about how logs are distributed between the `user` and `operator` Grafana.
I see a good example here https://github.com/gardener/gardener/blob/606a6af1ecafc16a99fedd0db2cb55534dd3d3f6/charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-configmap.yaml#L289-L297

Reply:
Seems that discriminating by record type is already obsolete or at least not in use anymore. Now records go to dedicated shoot Grafanas with predefined views on it depending on the role. Will skip.

Reply:
>  discriminating by record type is already obsolete or at least not in use anymore

I think it is wrong statement. @vlvasilev @Kristian-ZH, please check.

Reply:
statement coming exactly from them

Reply:
Yes, this is leftover from the `searchguard` plugin. Currently we do not use them and will remove them

Reply:
As discussed, added few lines dedicated to what you need to do as of today, should you want to change what's shown in the grafanas.